### PR TITLE
Iter4/eh profile picture

### PR DESCRIPTION
**Pull request by hunz-dev**

### CODE REVIEW

File: code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/ProfileActivity.java [hunk 1]
```diff
@@ -321,4 +332,19 @@ private void save()
         }
 
     }
+
+    private boolean validate() {
+        boolean isValid = true;
+        String url = mPictureView.getText().toString();
+
+        if (url.length() > 0)
+            isValid = Patterns.WEB_URL.matcher(url).matches();
```
Comment (MattMcMurray) [inline code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/ProfileActivity.java:341]: @hnz-evn does this check if it's an imgur URL specifically? Or just that it's a URL in general?

Comment (hunz-dev) [inline code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/ProfileActivity.java:341]: @MattMcMurray -- Any URL. I went under the assumption that the user could use any image hosting service. Same goes for the front-end validation on the web app.

File: code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/helpers/JsonConverter.java [hunk 2]
```diff
@@ -52,10 +52,14 @@ public static User userFromJson(JSONObject json) throws JSONException {
         if (json.has("helpDescription")) {
             helpDescription = json.getString("helpDescription");
         }
+        String picture = "";
+        if (json.has("helpDescription")) {
```
Comment (sbairos) [inline code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/helpers/JsonConverter.java]: Should be has("picture") not helpDescription

File: code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/ProfileActivity.java (hunk 1 above)
Comment (MattMcMurray) [inline code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/ProfileActivity.java:341]: Ah cool, that's awesome.

File: code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/helpers/JsonConverter.java (hunk 2 above)
Comment (MattMcMurray) [inline code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/helpers/JsonConverter.java]: @sbairos @hnz-evn fixed this in upcoming commit

**Pull request merged**